src/components/TestToolsModal.tsx

```diff
@@ -24,10 +25,13 @@ function getRouteBasedOnAuthStatus(isAuthenticated: boolean, activeRoute: string
     return isAuthenticated ? ROUTES.SETTINGS_CONSOLE.getRoute(activeRoute) : ROUTES.PUBLIC_CONSOLE_DEBUG.getRoute(activeRoute);
 }
 
+const modalContentMaxHeightPercentage = 0.75;
```

Comment:
Shall we move this const to variables?

Reply:
That makes sense, but I see Variables as less of repeatable const and more as a lib for tracking hardcoded design values. ie. sizes for images might only be used 1x but we store in there. How do you see it?